example-component/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include pentagoncomponent/templates/ *
```

Comment:
Is there a reason why this shouldn't have a trailing newline? Having one makes working with CLI tools a bit easier as (unfortunately) they were built expecting files to end with a newline.